src/pages/coffee.js

```diff
@@ -0,0 +1,26 @@
+import {products} from '../../backend/db';
+import FilterBar from '../components/FilterBar';
+import Header from '../components/Header';
+import ProductCard from '../components/ProductCard';
+import StyledProductList from '../components/ProductList/styled';
+
+export default function showCoffee() {
```

Comment:
This is a component and should start with an upper case letter!
```suggestion
export default function ShowCoffee() {
```